auth/README.md

```diff
@@ -146,6 +147,67 @@ allprojects {
 }
 ```
 
+#### GitHub
```

Comment:
Thanks for the detailed guide!  Could you add a bit more "why"?  Like describing why you need asset links and bits like that.